CafeApp/CafeApp/AddRestaurant.cs

```diff
@@ -36,17 +36,30 @@ public AddRestaurant( string email )
 
         private void AddButton_Click(object sender, EventArgs e)
         {
+            bool mandatoryFieldsEmpty = false;
+
+            //Mandatory to fill: name, address (,tables)
             name = NameBox.Text;
             address = AddressBox.Text;
             phone = PhoneBox.Text;
-            tables = int.Parse(TablesBox.Text);
+            tables = (int)TablesBox.Value;
```

Comment:
It is now impossible to input non-number value into TablesBox
So there is no need for Parsing